server/scripts/stats.js

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable */
+const { dbClient } = require('../db/db.js');
+const { collections } = require('../db/schema.js');
+const inquirer = require('inquirer');
+const dayjs = require('dayjs');
+const utc = require('dayjs/plugin/utc');
+require('dotenv').config({ path: '../.env' });
+
+dayjs.extend(utc);
+
+const toPostgresDate = (date) => {
+  return dayjs(date).utc().format('YYYY-MM-DD HH:mm:ssZ');
+}
+
+/* eslint-disable no-console */
+(async () => {
+  if (require.main === module) {
+    try {
+      await dbClient.connect();
+
+      const healthAuthorities = [
+        'Interior',
+        'Fraser',
+        'Vancouver Coastal',
+        'Vancouver Island',
+        'Northern',
+      ];
+
+      let startDate;
+      let endDate;
+
+      ({ 'Start Date': startDate } = await inquirer.prompt([{
+        name: 'Start Date',
+        type: 'input',
+        default: dayjs().format('YYYY-MM-DD')
+      }]));
+
+      ({ 'End Date (not inclusive)': endDate } = await inquirer.prompt([{
+        name: 'End Date (not inclusive)',
```

Comment:
Just making sure I understand this correctly - if I wanted to search records submitted today (2020-10-22) I would enter 2020-10-22 as my start date and 2020-10-23 as my end date?

Reply:
That's right. The end date already has the next day as default, so you just need to hit enter in this step.